src/txn/txn.c

```diff
@@ -716,8 +722,46 @@ __wt_txn_commit(WT_SESSION_IMPL *session, const char *cfg[])
 		WT_ERR_MSG(session, EINVAL,
 		    "commit_timestamp is required for a prepared transaction");
 
+#ifdef HAVE_DURABLE_TIMESTAMPS
+	/*
+	 * Durable timestamp is required for a prepared transaction.
+	 */
+	if (prepare) {
+		/*
+		 * Look for a durable timestamp.
```

Comment:
This comment should be on a single line. Actually, I don't think it's necessary because of the one above the `if (prepare)`